### DISCUSSION TITLE
If the Cancel button is pressed during C-Store transmission, I want to interrupt the transmission to the server.

### DISCUSSION THREAD

**Discussion by tomoguti** [fo-dicom/fo-dicom#1347] [Q&A]
Hi, I am currently developing a PACS export in fo-dicom and have run into a problem.
That is, if a cancel button is pressed while sending a DICOM file to a DICOM server, I want to send an A-ABORT-RQ to abort the sending to the server, but I do not know how to do that.
Can I just use AAbort() to create a PDU and send it to the server? However, I do not know how to send it.
Or should I use Cancellation?
I spent a day researching and could not find any samples, etc.

This text was created at DeepL. Sorry if it is not clear.

**Comment by gofal** [comment]
As I can read from code, sending an A-Abort is currently supported on server side. so if you are the SCP and you want to abort the client to send you further data, then you can call SendAbort. But from client-side this is not supported. To be more exactly: of course internally the client uses the same DicomConnection methods as the server, but the methods are not accessible via the IDicomClient interface.
